clusters/hive/pools/cvp/install-config-aws-us-east-1_secret.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: install-config-aws-us-east-1
+  namespace: cvp-cluster-pool
+stringData:
+  # The public key is taken from https://github.com/openshift/release/blob/80d88fb8ab7eec427d001ed7fb67741251c49dcf/core-services/ci-secret-bootstrap/_config.yaml#L869-L871
+  install-config.yaml: |
+    apiVersion: v1
+    baseDomain: hive.aws.ci.openshift.org
```

Comment:
We definitely do not want you to use our base domain.

Reply:
Is there a list of domain we can pick from? Or how do we pick the correct baseDomain? Reading through documentation, I dont know how to choose the correct one.

Reply:
This is not a selection, this is a part of the configuration you need to do on the AWS account. This is something that you need to configure. See https://docs.ci.openshift.org/docs/how-tos/cluster-claim/#prerequisites (second bullet) which will take you to https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-account.html#installation-aws-route53_installing-aws-account

Reply:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/optional-operators/cvp-common/aws/optional-operators-cvp-common-aws-workflow.yaml#L6